### ISSUE TITLE
Blog / News authors for theodi.org (staff member archived)

### ISSUE THREAD

**Issue by langphil** [theodi/shared#929]
The issue here is that when a staff member of theODI leaves the organisation their 'Person' artefact is archived in Panopticon. 

This causes an issue with blog posts that were assigned to these individuals as 'Authors'. The lack of author on the page means that the page returns a 'Page Error'.

What I would like is for the pages to continue to function after an author is archived and to maintain the name of the individual who authored the post. 

cc @Floppy & @pezholio

**Comment by Floppy** [comment]
Really, as the person still exists, should they not just have their bio updated and be removed from the "team" tag? Don't break the web, and all that...

**Comment by langphil** [comment]
Yes, that does sense. This is relates to individuals who were archived
before Giles created the tagging functionality on the team page.

-- 
*Phil Lang *
*Digital Production Editor*
*theodi.org* <http://theodi.org>

@langphil <https://twitter.com/langphil> | 0741 44 63047 <//0741 44 63047>

News How three big businesses create value with open innovation
<http://theodi.org/open-enterprise-big-business>

News ODI response: UK Land Registry privatisation consultation
<http://theodi.org/news/odi-responds-uk-land-registry-privatisation-consultation>

News ODI response: Google deep in trust issues around use of UK patient data
<http://blogs.bmj.com/bmj/2016/05/10/ellen-broad-and-tom-sasse-google-deep-in-trust-issues-around-use-of-uk-patient-data/>

On 4 July 2016 at 12:34:17, James Smith (notifications@github.com) wrote:

Really, as the person still exists, should they not just have their bio
updated and be removed from the "team" tag? Don't break the web, and all
that...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/theodi/shared/issues/929#issuecomment-230270449>, or mute
the thread
<https://github.com/notifications/unsubscribe/AEzAE3KjnaRIO5zgUswTOvbR8CR1Nribks5qSO-4gaJpZM4JEShe>
.
